docs/logic-apps/authentication.md

```diff
@@ -39,3 +40,30 @@ using (var logicApp = await LogicAppClient.CreateAsync(resourceGroup, logicAppNa
 {	
 }
 ```
+## Using a AccessToken
+
+The service principal used to request a token will need to have at least [`Logic App Contributor`](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#logic-app-contributor) permissions on all your Azure Logic App instances that you want to test.
```

Comment:
Duplicate of other auth option, let's move it before the auth details and list it as a pre-req

Reply:
LGTM, thanks!